_config.yml

```diff
@@ -1,7 +1,7 @@
 title: Wired Brain Coffee
 logo: "/images/company-logo.png?raw=true"
 description: >
-  High quality, slow roasted, premium beans for a slow balanced taste and rick aroma.
+  High quality, slow roasted, premium beans for a slow balanced taste and rich aroma.
```

Comment:
- [x] Finish my changes
- [ ] Push my commits to GitHub
- [ ] Open a pull request